### PR TITLE
aodn/backlog#2730 valid method checks RLS/ATRC

### PR DESCRIPTION
**Pull request by jonescc**
For https://github.com/aodn/backlog/issues/2730 ATRC and RLS Method valid checks

RLS:

image

ATRC:

image

### CODE REVIEW

File: api/src/main/java/au/org/aodn/nrmn/restapi/util/ValidatorHelpers.java [hunk 1]
```diff
@@ -22,14 +22,16 @@
...
     }
-  public <E,T> List<E> toErrorList(Validated<E, T>  validator) {
+
+    public static <E,T> List<E> toErrorList(Validated<E, T>  validator) {
         return  validator.bimap(Seq::of, Function.identity()).foldInvalidLeft(Monoids.seqConcat()).toList();
     }
 
 
-    public List<ErrorInput> reducetoErrrors(Validated<ErrorInput, String>  ...validators){
+    public static List<ErrorInput> reducetoErrrors(Validated<ErrorInput, String>  ...validators){
```
Comment (utas-tajona) [inline api/src/main/java/au/org/aodn/nrmn/restapi/util/ValidatorHelpers.java:31]: if you change this method it would be good to Generify ErrorInput to E

**Review by utas-tajona — APPROVED** [review]
beside the comment it looks good

File: api/src/main/java/au/org/aodn/nrmn/restapi/util/ValidatorHelpers.java (hunk 1 above)
Comment (jonescc) [reply api/src/main/java/au/org/aodn/nrmn/restapi/util/ValidatorHelpers.java:31]: Sure, no problem.

Comment (jonescc) [reply api/src/main/java/au/org/aodn/nrmn/restapi/util/ValidatorHelpers.java]: Done

**Pull request merged**